scripts/js/upgrade/index.js

```diff
@@ -0,0 +1,330 @@
+          if (result.status.isInBlock) {
+            console.log(
+                `Transaction included at blockHash ${result.status.asInBlock}`
+            );
+            resolve()
+          } else if (result.isError) {
+            console.log(`Transaction Error`);
+            reject("blabla bad")
+          }
+        });
+  });
+}
+
+async function councilCloseProposal(api, account, proposalHash, proposalIndex, nonce) {
+  return new Promise((resolve, reject) => {
+    console.log(
+        `--- Submitting extrinsic to close council motion. (nonce: ${nonce}) ---`
+    );
+
+    api.tx.council.close(proposalHash, proposalIndex, 52865600000, 82)
+        .signAndSend(account, {nonce: nonce, era: 0}, (result) => {
+          console.log(`Current status is ${result.status}`);
+          if (result.status.isInBlock) {
+
+            console.log(
+                `Transaction included at blockHash ${result.status.asInBlock}`
+            );
+
+            let democracyIndex;
+            result.events.forEach((er) => {
+              if (er.event.method === "Started") {
+                democracyIndex = er.event.data[0]
+              }
+            })
+
+            resolve(democracyIndex)
+          } else if (result.isError) {
+            console.log(`Transaction Error`);
+            reject("blabla bad")
+          }
+        });
+  });
+}
+
+async function voteReferenda(api, account, refIndex, nonce) {
+  return new Promise((resolve, reject) => {
+    console.log(
+        `--- Submitting extrinsic to vote on referenda. (nonce: ${nonce}) ---`
+    );
+
+    let vote = {
+      Standard: {
+        vote: {
+          aye: true,
+          conviction: 0
+        },
+        balance: "1000000000000000000"
+      }
+    }
+
+    api.tx.democracy.vote(refIndex, vote)
+        .signAndSend(account, {nonce: nonce, era: 0}, (result) => {
+          console.log(`Current status is ${result.status}`);
+          if (result.status.isInBlock) {
+            console.log(
+                `Transaction included at blockHash ${result.status.asInBlock}`
+            );
+            resolve()
+          } else if (result.isError) {
+            console.log(`Transaction Error`);
+            reject("blabla bad")
+          }
+        });
+  });
+}
+
+async function enactUpgrade(api, account, wasmCode, nonce) {
+  return new Promise((resolve, reject) => {
+    console.log(
+        `--- Submitting extrinsic to enact upgrade. (nonce: ${nonce}) ---`
+    );
+
+    api.tx.parachainSystem.enactAuthorizedUpgrade(wasmCode)
+        .signAndSend(account, {nonce: nonce, era: 0}, (result) => {
+          console.log(`Current status is ${result.status}`);
+          if (result.status.isInBlock) {
+            console.log(
+                `Transaction included at blockHash ${result.status.asInBlock}`
+            );
+            resolve()
+          } else if (result.isError) {
+            console.log(`Transaction Error`);
+            reject("blabla bad")
```

Comment:
Shall we print the `result` to have more info on what went wrong? Also, does "blabla" stand for something? 😆